modules/apps/translation/translation-test/src/testFunctional/tests/TranslationsManager.testcase

```diff
@@ -0,0 +1,108 @@
+
+	property osgi.modules.includes = "frontend-js-components-sample-web";
+	property portal.acceptance = "true";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.component.names = "Translation Manager";
+	property testray.main.component.name = "User Interface";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		JSONLayout.addPublicLayout(
+			groupName = "Guest",
+			layoutName = "JS Components Sample Page");
+
+		JSONLayout.addWidgetToPublicLayout(
+			groupName = "Guest",
+			layoutName = "JS Components Sample Page",
+			widgetName = "JS Components Sample");
+
+		Navigator.gotoPage(pageName = "JS Components Sample Page");
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			User.logoutPG();
+
+			JSONLayout.deletePublicLayout(
+				groupName = "Guest",
+				layoutName = "JS Components Sample Page");
+		}
+	}
+
+	@description = "LPS-147499. Verifies default language is available and views no other languages options."
+	@priority = "3"
+	test DefaultLanguageIsAvailableAndViewsNoOtherLanguagesOptions {
+		task ("Select translation language menu") {
+			Translations.viewSelectTranslationLanguageMenu(
+				key_currentLocale = "en-US",
+				key_title = "Admin");
+		}
+
+		task ("Check if element exists ") {
+			AssertElementPresent(
+				key_locale = "en-US",
+				locator1 = "Translation#DROPDOWN_MENU_ITEM");
+
+			AssertElementPresent(
+				key_locale = "ca-ES",
+				locator1 = "Translation#DROPDOWN_MENU_ITEM");
+
+			AssertElementPresent(
+				key_locale = "fr-FR",
+				locator1 = "Translation#DROPDOWN_MENU_ITEM");
+
+			AssertElementNotPresent(
+				key_locale = "zh-CN",
+				locator1 = "Translation#DROPDOWN_MENU_ITEM");
+
+			AssertElementNotPresent(
+				key_locale = "es-ES",
+				locator1 = "Translation#DROPDOWN_MENU_ITEM");
+
+			AssertElementPresent(locator1 = "Button#MANAGE_TRANSLATIONS");
+		}
+
+		task ("Assert link to Manage Translations is available") {
+			Click(locator1 = "Button#MANAGE_TRANSLATIONS");
+
+			AssertElementPresent(locator1 = "Portlet#MODAL_TITLE");
+		}
+	}
+
+	@description = "LPS-147612. Verifies translation results displayed side by side."
+	@priority = "3"
+	test TranslationResultsDisplayedSideBySide {
+		task ("Input different languages with the same word") {
+			Translations.editTranslationInput(
+				key_currentLocale = "en-US",
+				key_locale = "fr-FR",
+				key_title = "Admin",
+				languagefirst = "Welcome to Liferay",
+				languagesecond = "Bienvenue sur Liferay");
+		}
+
+		task ("View translation results displayed side by side") {
+			AssertElementPresent(
+				key_title = "Admin",
+				locator1 = "TextInput#INPUT_TEXT_FIELD",
+				value1 = "Bienvenue sur Liferay");
+
+			AssertElementPresent(
+				key_title = "Admin",
+				locator1 = "TextInput#INPUT_FORM_TEXT",
+				value1 = "Welcome to Liferay");
+		}
```

Comment:
Sorry, I don't quite understand what we're trying to test here. It looks like we're just inputting text into a couple forms and asserting that they're present? I think maybe `editTranslationInput` and the task `Input different languages with the same word` and  description `Verifies translation results displayed side by side.` Don't tell me enough to understand what's going on. I tried to read the Given When Then in https://issues.liferay.com/browse/LPS-147612 but it also doesn't make sense to me... Given When Given sequence doesn't make sense.

Could someone explain or show me a video of what we're trying to test? cc/ @zhangyan0701 @Hijky

Reply:
I reviewed the original test scenario which involves web content. This is what I found which looks different than what's available on js components sample portlet. To me, this is not testable with current js components sample portlet.
![image](https://user-images.githubusercontent.com/35235266/155220631-bd72cc71-0bf1-4c07-9a8b-bbf831186cba.png)

https://user-images.githubusercontent.com/35235266/155219370-d8654186-a519-4f06-b1cd-96ff0cc53bb5.mp4

cc/ @edalgrin @tinycarol @dsanz, is the side by side UI/feature something we will support for Translation Manager for all product teams? imo, it looks like an implementation only with Web Content and is outside the scope of testing for Translation Manager on FI side.

Reply:
The side-by-side is out of the scope of this module. That's a very particular use case where a variant of translation manager which does not sync language selections could be used, but we're not there yet.

Reply:
Thanks @dsanz ! 

@Hijky , please remove this test case from the in progress PR.

Reply:
ok, I will remove it.